src/sentry/integrations/msteams/webhook.py

```diff
@@ -103,7 +105,7 @@ def verify_signature(request):
     # get the open id config and jwks
     client = MsTeamsJwtClient()
     open_id_config = client.get_open_id_config()
-    jwks = client.get_cached(open_id_config["jwks_uri"])
+    jwks = cast(Mapping[str, Any], client.get_cached(open_id_config["jwks_uri"]))
```

Comment:
this cast is annoying -- but it's because our api clients aren't typesafe

Reply:
Yep. `get_cached` is actually typed as `Union[BaseApiResponse, Mapping[str, Any], Response]`, so that's fun.